### PR TITLE
Perf/eos evm

### PR DESCRIPTION
**Pull request by leisterfrancisco**
### What does this PR do?

- Create logic to enable env for EOSIO (jungle, eos)

### CODE REVIEW

File: .github/workflows/deploy-jungle-testnet.yaml [hunk 1]
```diff
@@ -106,6 +107,25 @@ jobs:
...
+          HAPI_EVM_NETWORK_CHAIN_ID: 73e4385a2708e6d7048834fbc1079f2fabb17b3c125b146af438971e90716c4d
+          HAPI_EVM_EOS_EVM_ACCOUNT: eosio.evm
+          HAPI_EVM_BLOCK_INTERVAL_SEC: 1
+          HAPI_EVM_OLD_BLOCK_INTERVAL_SEC: 0.1
+          HAPI_EVM_ATH_INTERVAL_SEC: 60
+          HAPI_EVM_CLEAN_OLD_BLOCK_INTERVAL_SEC: 86400
+          HAPI_EVM_CLEAN_OLD_TRANSFER_INTERVAL_SEC: 86400
+          HAPI_EVM_KEEP_HISTORY_FOR_YEARS: 1
+          HAPI_EVM_HYPERION_API: https://jungle.eosusa.io
+          HAPI_EVM_HYPERION_START_AT: 2022-01-01T00:00:00.000+00:00
```
Comment (leisterfrancisco) [inline .github/workflows/deploy-jungle-testnet.yaml:128]: @Torresmorah can you help me verifying that these values are correct?

File: .github/workflows/deploy-mainnet.yaml [hunk 2]
```diff
@@ -104,7 +105,25 @@ jobs:
...
+          HAPI_EVM_NETWORK_CHAIN_ID: aca376f206b8fc25a6ed44dbdc66547c36c6c33e3a119ffbeaef943642f0e906
+          HAPI_EVM_EOS_EVM_ACCOUNT: eosio.evm
+          HAPI_EVM_BLOCK_INTERVAL_SEC: 1
+          HAPI_EVM_OLD_BLOCK_INTERVAL_SEC: 0.1
+          HAPI_EVM_ATH_INTERVAL_SEC: 60
+          HAPI_EVM_CLEAN_OLD_BLOCK_INTERVAL_SEC: 86400
+          HAPI_EVM_CLEAN_OLD_TRANSFER_INTERVAL_SEC: 86400
+          HAPI_EVM_KEEP_HISTORY_FOR_YEARS: 1
+          HAPI_EVM_HYPERION_API: https://api.eossweden.org
+          HAPI_EVM_HYPERION_START_AT: 2022-01-01T00:00:00.000+00:00
```
Comment (leisterfrancisco) [inline .github/workflows/deploy-mainnet.yaml:126]: @Torresmorah can you help me verifying that these values are correct?

File: .github/workflows/deploy-jungle-testnet.yaml (hunk 1 above)
Comment (Torresmorah) [reply .github/workflows/deploy-jungle-testnet.yaml:128]: yes, the values are correct

File: .github/workflows/deploy-mainnet.yaml (hunk 2 above)
Comment (Torresmorah) [reply .github/workflows/deploy-mainnet.yaml:126]: yes, the values are correct, but the hyperion API is not working

File: .github/workflows/deploy-jungle-testnet.yaml
Comment (Torresmorah) [inline .github/workflows/deploy-jungle-testnet.yaml:1]: we need to add this variable
HASURA_GRAPHQL_ACTION_EVM_URL: http://dashboard-hapi-evm:9091

File: .env.local
Comment (Torresmorah) [inline .env.local:1]: I think the change should be applied in the .env.mainnet file

File: .github/workflows/deploy-mainnet.yaml (hunk 2 above)
Comment (leisterfrancisco) [reply .github/workflows/deploy-mainnet.yaml]: Fixed, thanks

File: .github/workflows/deploy-jungle-testnet.yaml
Comment (leisterfrancisco) [reply .github/workflows/deploy-jungle-testnet.yaml:1]: Done, thank you.

File: .env.local
Comment (leisterfrancisco) [reply .env.local:1]: done

File: .github/workflows/deploy-mainnet.yaml [hunk 3]
```diff
@@ -104,13 +105,32 @@ jobs:
           HAPI_EOSRATE_GET_STATS_URL: ${{ secrets.HAPI_EOSRATE_GET_STATS_URL }}
           HAPI_EOSRATE_GET_STATS_USER: ${{ secrets.HAPI_EOSRATE_GET_STATS_USER }}
           HAPI_EOSRATE_GET_STATS_PASSWORD: ${{ secrets.HAPI_EOSRATE_GET_STATS_PASSWORD }}
-
+          # hapi-evm
+          HAPI_EVM_SERVER_PORT: '9090'
+          HAPI_EVM_SERVER_ADDRESS: '0.0.0.0'
+          HAPI_EVM_HASURA_URL: 'http://dashboard-hasura:8080/v1/graphql'
+          HAPI_EVM_HASURA_ADMIN_SECRET: ${{ secrets.HAPI_EVM_HASURA_ADMIN_SECRET }}
+          HAPI_EVM_DATABASE_URL: ${{ secrets.HAPI_EVM_DATABASE_URL }}
```
Comment (xavier506) [inline .github/workflows/deploy-mainnet.yaml:113]: @leisterfrancisco we will need to set this secret in github, can you send the value needed over slack please

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**